source/views/building-hours/report/overview.js

```diff
@@ -192,7 +217,7 @@ export class BuildingHoursProblemReportView extends React.PureComponent {
             />
           </Section>
 
-          <Section footer="Thanks for reporting!">
+          <Section>
```

Comment:
I would like it for now, at least. I do want to rethink our buttons in this view, though, but that's a matter for a different PR.